<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newwindow</class>
 <widget class="QWidget" name="newwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="heroStats2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>101</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="item_btn_class" name="item_btn_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="item_btn_class" name="item_btn_1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="return_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="xp_bar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
   <property name="format">
    <string>%v / %m xp</string>
   </property>
  </widget>
  <widget class="QLabel" name="hero_lvl_lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="infobox_lbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>111</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="item_btn_class" name="current_item_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="item_btn_class" name="current_item_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>item_btn_class</class>
   <extends>QPushButton</extends>
   <header location="global">item_btn_class.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
